src/scripting/game_lua_kernel.cpp

```diff
@@ -2635,12 +2636,12 @@ int game_lua_kernel::intf_set_floating_label(lua_State* L, bool spawn)
 			loc = luaW_checklocation(L, -1);
 		}
 		if(luaW_tableget(L, 2, "halign")) {
-			static const char* options[] = {"left", "center", "right"};
-			alignment = font::ALIGN(luaL_checkoption(L, -1, nullptr, options));
+			static constexpr auto options = []() { return std::array{"left", "center", "right"}; }();
```

Comment:
No need for an IIFE here. Just declare the `std::array` directly. Likewise below.